baseplate.go

```diff
@@ -20,7 +20,19 @@ import (
 	"github.com/reddit/baseplate.go/tracing"
 )
 
-// Config is a general purpose config for assembling a Baseplate server
+// Configer defines the interface that allows you to extend Config with your own
+// configurations.
+type Configer interface {
+	GetConfig() Config
```

Comment:
:no_bell: just
```go
Config() Config
```
?

Reply:
this won't work for the structs embedding it (because the field is also named `Config`, because embedding)